kod/object/active/holder/nomoveon/battler/player/user/dm/admin.kod

```diff
@@ -138,12 +139,15 @@ messages:
 
    UserSay(string = $,type = $)
    {
-      local oWeapon, i, oObject;
+      local oWeapon, i, oObject, lItemAttributes, lAttribute, bShouldRevealAttribute;
 
       if type = SAY_DM
          AND (StringContain(string,admin_create_itematt_command)
-              OR StringContain(string,"create itematt"))
+              OR StringContain(string,"create itematt")
+              OR StringContain(string, admin_create_unrevealed_itematt_command))
       {
+         bShouldRevealAttribute = NOT StringContain(string, admin_create_unrevealed_itematt_command);
```

Comment:
This is still also being checked in the previous line, too.  The idea was to check it once.

Reply:
Done, should be good now.